test/JDBC/input/BABEL-3507-vu-prepare.sql

```diff
@@ -0,0 +1,112 @@
+-- max values causing overflow in avg (BIGINT,INT) 
+CREATE TABLE babel_3507_vu_prepare_t1(
+avgbigint BIGINT, avgint INT , avgsmallint SMALLINT , avgtinyint TINYINT )
+GO
+
```

Comment:
we should also add tests for negative/positive values mix, NULL values, divide by zero, result being zero

Reply:
Added extra tests in the latest commit